terraform/environments/oasys-national-reporting/locals_test.tf

```diff
@@ -31,26 +31,44 @@ locals {
       }
     }
 
-    # baseline_ec2_instances = {
-    #   test-db = merge(local.defaults_onr_db_ec2 ,{
-    #     config = merge(local.defaults_onr_db_ec2.config, {
-    #       ami_name          = "hmpps_ol_8_5_oracledb_19c_release_2024-*"
-    #       availability_zone = "${local.region}a"
-    #     })
-    #     instance = merge(local.defaults_onr_db_ec2.instance, {
-    #       instance_type                = "r6i.xlarge"
-    #       metadata_options_http_tokens = "optional"
-    #     })
-    #     user_data_cloud_init = module.baseline_presets.ec2_instance.user_data_cloud_init.ssm_agent_and_ansible
-    #     secretsmanager_secrets = module.baseline_presets.ec2_instance.secretsmanager_secrets.oracle_19c
-    #     tags = merge(local.defaults_onr_db_ec2.tags, {
-    #       ami         = "base_ol_8_5"
-    #       server-type = "onr-db-test"
-    #       oracle-sids = "T3ONRAUD T3ONRBDS T3ONRSYS"
-    #       description = "ONR db for BOE Enterprise XI INSTALLATION TESTING ONLY"
-    #     })
-    #   })
-    # }
+    baseline_ec2_instances = {
+      tt-onr-bods-1-a = merge(local.defaults_bods_ec2, {
+        config = merge(local.defaults_bods_ec2.config, {
+          availability_zone             = "${local.region}a"
+          ebs_volumes_copy_all_from_ami = false
+          user_data_raw                 = module.baseline_presets.ec2_instance.user_data_raw["user-data-pwsh"]
+        })
+        instance = merge(local.defaults_bods_ec2.instance, {
+          instance_type = "t3.large"
```

Comment:
make this m4.xlarge off the bat - this is the size/type we're going to use for "test" env anyway